src/screens/WalletConnect/WalletConnectPinConfirm.js

```diff
@@ -184,19 +185,23 @@ class WalletConnectPinConfirmScreeen extends React.Component<Props, State> {
 
   render() {
     const { isChecking } = this.state;
+    const { useBiometrics } = this.props;
     return (
       <CheckAuth
         onPinValid={this.handleCallRequest}
         isChecking={isChecking}
         headerProps={{ onBack: this.handleBack }}
+        enforcePin={!useBiometrics}
       />
     );
   }
 }
 
 const mapStateToProps = ({
   walletConnect: { requests },
+  appSettings: { data: { useBiometrics = false } },
```

Comment:
Maybe instead of putting the default param here, it's better to use the appSettings migration when we load the data on app start?